src/lib/LavalinkNode.ts

```
@@ -1,264 +1,489 @@
+		switch (msg.op) {
+			case LavalinkOPTypes.Ready:
+				if (msg.sessionId) this.sessionId = msg.sessionId;
+				if (!this._sessionUpdated) {
+					this._sessionUpdated = true;
+					Rest.updateSession(this).catch((e) => this.manager.emit("error", e, this));
+				}
+				break;
+
+			case LavalinkOPTypes.Stats: {
+				const stats = { ...msg } as Stats;
+				delete (stats as { op?: number }).op;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)